docs/components/Flex/Flex.stories.mdx

```diff
@@ -25,4 +23,6 @@ further control the layout of the children.
 ## Related Components
```

Comment:
One other docs thought, and I know we're just re-using the same example from mobile, but in the "nested" example even after playing with the props I'm still not 100% sure which elements are actually being adjusted other than I can see the space between the `arrowRight` icon and the rest of the icon changing, or stacking on `direction=column`. Either a simpler example, or some kind of visual indicator (custom background color?) to show which portion is being acted on by the props would make it easier to parse how the props are acting.

![image](https://github.com/GetJobber/atlantis/assets/39704901/c435fe8c-42ad-4175-8f9b-4bf1742cb29a)

Reply:
> Minor and probably a followup, but adding Content as another related component for even simpler layouts might be helpful

Agreed. This would be a great addition as a fast follow.

>Either a simpler example, or some kind of visual indicator (custom background color?) to show which portion is being acted on by the props would make it easier to parse how the props are acting.

Funny you mention that because i considered exactly that. I'm still mulling over the idea of background colours just to explain where stuff is going.